qiskit/dagcircuit/dagcircuit.py

```diff
@@ -295,13 +304,8 @@ def _add_wire(self, wire):
         if wire not in self._wires:
             self._wires.add(wire)
 
-            try:
-                wire_name = "%s[%s]" % (wire.register.name, wire.index)
-            except CircuitError:
-                wire_name = "%s" % wire
-
-            inp_node = DAGNode(type='in', name=wire_name, wire=wire)
-            outp_node = DAGNode(type='out', name=wire_name, wire=wire)
```

Comment:
I don't think anything uses it, except for the drawer, but this means we're dropping the name for i/o nodes right? What does the dag drawer output look like without a name?

Reply:
Nevermind about the dag drawer, I got to that later in the review and see you're building the name in the drawer directly so it'll look the same.